services/web-server/src/login/strategies/mozilla-auth0.js

```diff
@@ -240,6 +239,7 @@ module.exports = class MozillaAuth0 {
       (request, response) => {
         response.render('callback', {
           user: request.user,
+          publicUrl: cfg.app.publicUrl,
         });
```

Comment:
Would it be useful to abstract this out to a utility?

Reply:
I'm not sure I understand the question. What kind of utility?

Reply:
These five lines appear in both strategies, and likely would appear in all strategies.  It's a minor detail.